src/utils/presence.js

```diff
@@ -39,20 +39,30 @@ export const getAggregatedPresence = (presence: UserPresence): ClientPresence =>
    * If there are several ClientPresence objects with the greatest
    * PresenceStatus, an arbitrary one is chosen.
    */
+
+  let status = 'offline';
+  let client = '';
+  let timestamp = 0;
+
   Object.keys(presence)
-    .filter((client: string) => client !== 'aggregated')
-    .reduce(
-      (aggregated, client: string) => {
-        const { status, timestamp } = presence[client];
-        if (Date.now() / 1000 - timestamp < OFFLINE_THRESHOLD_SECS) {
-          if (presenceStatusGeq(status, aggregated.status)) {
-            return { client, status, timestamp };
-          }
+    .filter((c: string) => c !== 'aggregated')
+    .forEach(c => {
+      const devicePresence = presence[c];
+      const age = Date.now() / 1000 - devicePresence.timestamp;
+      if (timestamp < devicePresence.timestamp) {
+        /* eslint-disable prefer-destructuring */
+        timestamp = devicePresence.timestamp;
```

Comment:
That is kind of wacky that the lint rule wants to change this!

It looks like that's optional:
https://eslint.org/docs/rules/prefer-destructuring
and can be disabled with something like
```yaml
    prefer-destructuring:
    - error
    - AssignmentExpression: {array: false, object: false}
```
So maybe let's do that.